html/static/js/html.js

```diff
@@ -0,0 +1,89 @@
+/* Javascript for HTMLXBlock. */
+var CUSTOM_FONTS, STANDARD_FONTS, _getFonts;
+
+CUSTOM_FONTS = "Default='Open Sans', Verdana, Arial, Helvetica, sans-serif;";
+STANDARD_FONTS = "Andale Mono=andale mono,times;" + "Arial=arial,helvetica,sans-serif;" + "Arial Black=arial black,avant garde;" + "Book Antiqua=book antiqua,palatino;" + "Comic Sans MS=comic sans ms,sans-serif;" + "Courier New=courier new,courier;" + "Georgia=georgia,palatino;" + "Helvetica=helvetica;" + "Impact=impact,chicago;" + "Symbol=symbol;" + "Tahoma=tahoma,arial,helvetica,sans-serif;" + "Terminal=terminal,monaco;" + "Times New Roman=times new roman,times;" + "Trebuchet MS=trebuchet ms,geneva;" + "Verdana=verdana,geneva;" + "Webdings=webdings;" + "Wingdings=wingdings,zapf dingbats";
+
+_getFonts = function () {
+  return CUSTOM_FONTS + STANDARD_FONTS;
+};
+
+var PLUGINS, PLUGINS_DIR, _getExternalPlugins;
+
+PLUGINS = ["codesample", "image", "link", "lists", "textcolor", "codemirror"];
+PLUGINS_DIR = "/resource/html5/public/plugins/";
+
+_getExternalPlugins = function () {
+  var externalPlugins = {};
+
+  for (let plugin of PLUGINS) {
+    externalPlugins[plugin] = PLUGINS_DIR + plugin + "/plugin.min.js"
+  }
+  return externalPlugins
+};
+
+function HTML5XBlock(runtime, element) {
+  var editorChoice = $("#editor-tab").data("editor");
```

Comment:
Can we avoid using jQuery in this file? http://youmightnotneedjquery.com/

Reply:
I couldn't get rid of `$.ajax({});` it keeps returning `403 Forbidden` even when I sent the CSRF Tokens and the permissions.